### DISCUSSION TITLE
API nota fiscal

### DISCUSSION THREAD

**Discussion by yasron** [RamonSilva20/mapos#1932] [Q&A] [answered]
Olá pessoal, gostaria de saber se é possivel instalar uma api de nota fiscal no sistema, e se têm algum progrador interessado em fazer isso para mim, pois infelizmente sou apenas usuario do sistema e tenho essa necessidade de implementação.

**Comment by SuperPorte** [comment, marked as answer]
No grupo do telegram, indicaram esse contato:

`Procura o @‪+55 11 98562-5975‬ 
Ele tem a solução para o MAPOS`

Não cheguei a contactar

**Comment by kristina100** [comment]
您好，已经收到（系统自动回复）
